lib/reek/smells/smell_repository.rb

```diff
@@ -42,12 +50,20 @@ def examine(scope)
 
       private
 
+      attr_writer :detectors, :typed_detectors, :configuration, :source_via, :smell_types
+
+      def initialize_detectors
```

Comment:
I don't follow - in where? The getter or the setter? Both places don't make much sense to me...